l10n_be_vat_reports/wizard/l10n_be_vat_intra.py

```diff
@@ -316,6 +316,7 @@ def create_xml(self):
         :return: Value for next action.
         :rtype: dict
         """
+        # todo use OCA module report_xml
```

Comment:
this is not what you said you're doing in the PR description ? if yes you can remove this comment.

Reply:
ok I get it. Just one report goes by oca xml report